docs/en/stack/ml/nlp/ml-nlp-shared.asciidoc

```diff
@@ -0,0 +1,41 @@
+tag::nlp-eland-clone-docker-build[]
+You can use the {eland-docs}[Eland client] to install the {nlp} model. Eland 
+commands can be ran via Docker. First, you need to clone the Eland repository 
+then create a Docker image of Eland:
+
+[source,shell]
+--------------------------------------------------
+git clone git@github.com:elastic/eland.git
+cd eland
+docker build -t elastic/eland .
+--------------------------------------------------
+
+After the script finished, your Eland Docker client is ready to use.
+end::nlp-eland-clone-docker-build[]
+
+tag::nlp-requirements[]
+To follow along the process on this page, you must have:
+
+* an {es} cluster that is set up properly to use the {ml-features}. Refer to 
+<<setup>>.
+
+* The {subscriptions}[appropriate subscription] level or the free trial period 
+activated.
+
+* https://docs.docker.com/get-docker/[Docker] installed.
+end::nlp-requirements[]
+
+tag::nlp-start[]
+Since the `--start` option is used at the end of the Eland import command, {es} 
+deploys the model to all available {ml} nodes and load the model in memory. If 
+you have multiple models and want to select which model to deploy, you can use 
+the **{ml-app} > Model Management** user interface in {kib} to manage the start 
+and stopping of models.
+end::nlp-start[]
+
+tag::nlp-sync[]
+Go to the **{ml-app} > Trained Models** page and synchroinze your trained 
+models. A warning message is displayed at the top of the page that says 
```

Comment:
> A warning message...

I think the last time I played with this, it automatically synched for me after a certain interval.  For that reason, I worded it like this in the Deploy models page: https://www.elastic.co/guide/en/machine-learning/current/ml-nlp-deploy-models.html#ml-nlp-deploy-model

Reply:
The ML app syncs automatically once an hour. It's like waiting for a bus sometimes you get lucky and one arrives straight away